internal/elasticsearch/v7/client.go

```diff
@@ -161,3 +163,9 @@ func preparePayload(item *opencdc.Record) (json.RawMessage, error) {
 		return itemPayload.Bytes(), nil
 	}
 }
+
+// Search calls the elasticsearch search api and retuns SearchResponse read from an index.
+func (c *Client) Search(_ context.Context, _ *api.SearchRequest) (*api.SearchResponse, error) {
+	// TODO: implement elasticsearch search api
+	return nil, nil //nolint:nilnil // implementation to be done
```

Comment:
IMHO it's better to return an error saying the method is not implemented. Otherwise, the method might be called by mistake, and it returns a nil error meaning it succeeded.

Reply:
Agree, updated now.